ParabolInc/parabol
# chore(one-on-one): Add user picker

Partial solution for #8560

This PR introduces multi user picker with an ability to add a custom option

Styles and other things will be added in next PRs

https://github.com/ParabolInc/parabol/assets/466991/6541df2a-d0c0-497d-b254-4fcaca38ef69

**How to test:**

- Open one-on-one meeting page, see that user picker works
- See that not one-on-one checkin is still working fine
